src/extension/android/adb.ts

```diff
@@ -214,6 +214,7 @@ export class AdbHelper {
             // For Windows we need to unescape files separators and drive letter separators
             sdkLocation = sdkLocation.replace(/\\\\/g, "\\").replace("\\:", ":");
         }
+        sdkLocation = `"` + sdkLocation + `"` // Encase SDK location in quotes in case it contains a space
```

Comment:
Thanks for contributing, seems this cannot get correct adb path correctly, you can check method `const isExist = fs.existsSync(localPropertiesSdkPath);` in function `getAdbPath()` . The emulator can be opened since if the location is not existing, extension will use command `adb` to open with environment variable directly.

Reply:
This would not work on windows because backslash is a path component. Maybe ill replace spaces with a carrot and a space on the line above that only runs on windows, and add an else that replaces spaces with a backslash and a space.